### PR TITLE
Fyne/wasm

### PR DESCRIPTION
**Pull request by Bluebugs**
This should provide a working support for wasm. There is still some instability depending on your browser and OS configuration.

### CODE REVIEW

File: browser.go [hunk 1]
```diff
@@ -121,7 +123,8 @@ func CreateWindow(_, _ int, title string, monitor *Monitor, share *Window) (*Win
 			go w.framebufferSizeCallback(w, w.canvas.Width, w.canvas.Height)
 		}
 		if w.sizeCallback != nil {
-			go w.sizeCallback(w, int(w.canvas.GetBoundingClientRect().Width), int(w.canvas.GetBoundingClientRect().Height))
+			boundingW, boundingH := w.GetSize()
```
Comment (andydotxyz) [inline browser.go:126]: I don't know it very well - but isn't it a risk to move from the canvas bounding rect to the window size?

File: browser.go [hunk 2]
```diff
@@ -438,11 +453,10 @@ func (w *Window) SetFramebufferSizeCallback(cbfun FramebufferSizeCallback) (prev
 }
 
 func (w *Window) GetSize() (width, height int) {
-	// TODO: See if dpi adjustments need to be made.
-	fmt.Println("Window.GetSize:", w.canvas.GetBoundingClientRect().Width, w.canvas.GetBoundingClientRect().Height,
-		"->", int(w.canvas.GetBoundingClientRect().Width), int(w.canvas.GetBoundingClientRect().Height))
+	width = int(w.canvas.GetBoundingClientRect().Width*w.devicePixelRatio + 0.5)
```
Comment (andydotxyz) [inline browser.go:456]: What is 0.5 here?

**Review by andydotxyz — COMMENTED** [review]
Looks cool, just 2 questions about changes that seem non-obvious

File: browser.go (hunk 2 above)
Comment (Bluebugs) [reply browser.go:456]: It is a way to do rounding from float to int to the upper boundary when above .5 and the below boundary when below .5 for any number.

File: browser.go (hunk 1 above)
Comment (Bluebugs) [reply browser.go]: I don't know, but I can actually just use the Width and Height that has been calculated in this listener it seems.

**Comment by andydotxyz** [comment]
In the last 2 commits part of the work seems to be commenting out code.
If it is not needed any more please remove, if it should be re-added imminently then please put TODO or some explanation. I realise now that it is not in our contributor guidelines (I will add) but commented out code should not be added (unless with a good reason).

**Review by andydotxyz — APPROVED** [review]
Looks good to me, anyone else want to comment before merge?

**Pull request merged**